pdf/creator/utils.go

```diff
@@ -43,3 +44,21 @@ func loadPagesFromFile(path string) ([]*model.PdfPage, error) {
 
 	return pages, nil
 }
+
+func round(val float64, roundOn float64, places int) float64 {
```

Comment:
Can you add a comment on what exactly this does?  Would be useful for internal use even though not exported as not obvious.

Reply:
I reviewed the usage of this util function and removed it because it was not necessary.